<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaneAdresowe</class>
 <widget class="QMainWindow" name="DaneAdresowe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DaneAdresowe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Ikony.qrc">
    <normaloff>:/new/ikony/icons/address-book-new.svg</normaloff>:/new/ikony/icons/address-book-new.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="Ikony.qrc">:/new/ikony/icons/contact-new.svg</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="Ikony.qrc">:/new/ikony/icons/mail.svg</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="mailEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="Ikony.qrc">:/new/ikony/icons/phone.svg</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="phoneEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Ikony.qrc">:/new/ikony/icons/address-book-new.svg</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filenameEdit">
        <property name="text">
         <string>None</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Ikony.qrc">
     <normaloff>:/new/ikony/icons/document-new.svg</normaloff>:/new/ikony/icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Ikony.qrc">
     <normaloff>:/new/ikony/icons/document-open.svg</normaloff>:/new/ikony/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Ikony.qrc">
     <normaloff>:/new/ikony/icons/document-save.svg</normaloff>:/new/ikony/icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Ikony.qrc"/>
 </resources>
 <connections/>
</ui>
